selfdrive/car/honda/carcontroller.py

```diff
@@ -121,7 +121,7 @@ def update(self, sendcan, enabled, CS, frame, actuators, \
     if CS.CP.carFingerprint in (CAR.CIVIC, CAR.ODYSSEY):
       is_fw_modified = os.getenv("DONGLE_ID") in ['b0f5a01cf604185c']
       STEER_MAX = 0x1FFF if is_fw_modified else 0x1000
-    elif CS.crv:
+    elif CS.crv or CS.acura_rdx:
```

Comment:
this is a bug. Actually the same bug that broke that was not civic or odyssey. replace by:
```elif CS.CP.carFingerprint in (CAR.CRV, CAR.ACURA_RDX):```

Reply:
Fixed